resources/webroot/stats.mustache

```diff
@@ -0,0 +1,1507 @@
+                                    </div>
+                                </div>
+                                <div class="col-md-6 p-1">
+                                    <div class="list-group">
+                                        <a class="list-group-item valor">
+                                            <h3 class="pull-right"><img src="./static/img/team/2.png" width="64" height="64"/></h3>
+                                            <h4 class="list-group-item-heading valor-gyms-count">0</h4>
+                                            <p class="list-group-item-text">{{stats_valor_gyms}}</p>
+                                        </a>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col-md-6 p-1">
+                                    <div class="list-group">
+                                        <a class="list-group-item mystic">
+                                            <h3 class="pull-right"><img src="./static/img/team/1.png" width="64" height="64"/></h3>
+                                            <h4 class="list-group-item-heading mystic-gyms-count">0</h4>
+                                            <p class="list-group-item-text">{{stats_mystic_gyms}}</p>
+                                        </a>
+                                    </div>
+                                </div>
+                                <div class="col-md-6 p-1">
+                                    <div class="list-group">
+                                        <a class="list-group-item instinct">
+                                            <h3 class="pull-right"><img src="./static/img/team/3.png" width="64" height="64"/></h3>
+                                            <h4 class="list-group-item-heading instinct-gyms-count">0</h4>
+                                            <p class="list-group-item-text">{{stats_instinct_gyms}}</p>
+                                        </a>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <br>
+                    <div class="row justify-content-center container">
+                        <div class="col-md-12">
+                            <div class="card">
+                                <div class="card-header bg-dark text-light"><b>{{stats_new_gyms}}</b></div>
+                                <div class="card-body">
+                                    <ul class="list-group list-group-flush">
+                                        {{#new_gyms}}
+                                            <li class="list-group-item">
+                                                <a href="https://maps.google.com/maps?q={{lat}},{{lon}}" target="_blank">{{name}} {{id}}</a>
```

Comment:
@123FLO321 I think at this place it would be better to link to the map itself. How can I access the map url on this line? :)